components/DetailsCard.js

```diff
@@ -21,7 +21,12 @@ export default function DetailsCard({ matchingPlace }) {
         />
         <h3>{matchingPlace.name}</h3>
         <p>Region: {matchingPlace.region}</p>
-        <p>Activity: {matchingPlace.activity}</p>
+        <p>
+          Activity:{" "}
+          {Array.isArray(matchingPlace.activity)
+            ? matchingPlace.activity.join(", ")
+            : matchingPlace.activity}
```

Comment:
This will always be an array I think, so this should work just as well:

```suggestion
          {matchingPlace.activity.join(", ")}
```